src/js/popup.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+import { initDomainButton, setDomainEventListeners } from './domain.js';
+import { addHistory, setHistoryEventListeners } from './history.js';
+import { setScreenSwitchEventListeners } from './screenSwitch.js';
+import { initWhiteList, setWhiteListEventListeners } from './whitelist.js';
+
+const initKillOverWindow = () => {
+  chrome.storage.sync.get('tabKillerIsOverWindows', (items) => {
+    if (items.tabKillerIsOverWindows === undefined) {
+      chrome.storage.sync.set({ tabKillerIsOverWindows: false });
+    }
+    document.getElementById('target_all_windows').checked =
+      items.tabKillerIsOverWindows;
```

Comment:
`items.tabKillerIsOverWindows`が`undefined`のとき、チェックボックスに`undefined`が代入されることになっている。

```suggestion
    const isTabKillerIsOverWindowsUndefined =
      items.tabKillerIsOverWindows === undefined;
    if (isTabKillerIsOverWindowsUndefined) {
      chrome.storage.sync.set({ tabKillerIsOverWindows: false });
    }
    document.getElementById('target_all_windows').checked =
      isTabKillerIsOverWindowsUndefined ? false : items.tabKillerIsOverWindows;
```